README.md

```diff
@@ -1,32 +1,42 @@
-# SPITO
-SPTIO (Simple Powerful Interprocess Tool to Overpower) has been designed
-to allow users easily swap and edit configuration files.
+# Spito
+Spito has been designed to allow users easily swap 
+and edit configuration files. It's user-friendly and
+lays the foundations for auto-repair tools.
 
 ## Use cases
-**For a while it's a myth language**
+**Currently, we are working on these features**
 
-### Simple config
-- used rule: samba-guest-are-god
-- purpose: use whole ready-to-use samba config
+### Simple config - samba
+Samba is a simple FOSS file server. Samba is this kind of software
+that requires configuration on start, and it costs a lot of searching 
+when you want to do anything.
+
+**samba-guests-are-god:**
+This rule allows samba guests to do almost everything. It's especially 
+useful when using virtual machines without shared files support.
 ```shell
-spito -r samba-guests-are-god
+spito samba-guests-are-god
```

Comment:
```suggestion
spito Nasz-Elektryk/samba samba-guests-are-god
```
Remember that requiring rule in SPITO looks like e.g. `github.com/BaderBC/ruleset rule-name` where `github.com/` is optional

Do not forget that the same applies to other spito command examples bellow